src/tutorial/runtime/introduction.rst

```diff
@@ -291,60 +334,163 @@ Where Do All The Files Go?
 
          ~/cylc-run/<workflow-name>/log/job/<cycle-point>/<task-name>/<job-submission-num>/
 
-      .. image:: ../img/cylc-gui-view-log.png
-         :align: center
-         :scale: 75%
+      .. TODO REPLACE THIS IF APPROPRIATE
+
+         .. image:: ../img/cylc-gui-view-log.png
+            :align: center
+            :scale: 75%
 
 
-Running A Workflow
-------------------
+Installing A Workflow
+---------------------
 
 .. ifnotslides::
 
-   It is a good idea to check a workflow for errors before running it.
-   Cylc provides a command which automatically checks for any obvious
-   configuration issues called ``cylc validate``, run via:
+   .. seealso::
 
-.. code-block:: sub
+      :ref:`The full guide to Cylc install <Installing-workflows>`.
+
+   .. versionchanged:: 8.0.0
+
+      ``cylc install`` is a new feature of Cylc 8.
+
```

Comment:
Not sure we should bother with these for the tutorials; they are more useful for the reference section of the docs. Otherwise we will have inconsistency within the tutorials when it comes to labelling new or changed features.
```suggestion
```